kernelci.org/content/en/docs/legacy/instances/chromeos/_index.md

```diff
@@ -300,3 +300,17 @@ for examples how to install the images within the published `chromiumos_test_ima
 KernelCI maintains a [Changelog](../chromeos/chromeos_changelog/) which tracks the evolution
 between published ChromiumOS releases as well as divergences between the KernelCI
 images and ChromiumOS upstream.
+
+# Upstreaming ChromiumOS changes
+
+Sometimes we have to fix issues and push changes upstream to ChromiumOS to avoid divergence.
+[Here](https://chromium.googlesource.com/chromiumos/docs/+/HEAD/contributing.md) is the official contributing documentation.
```

Comment:
"here" links are not great, but that's not a blocker.  Just not best practice in documentation and hypertext as it doesn't have any meaning without the HTTP link.  Something like "Please take a look at the [official contributing documentation](http link) for more details" is better.  A simple guideline is to think whether it would work with text-to-speech.